src/sourmash/cli/utils.py

```diff
@@ -50,11 +50,34 @@ def add_ksize_arg(parser, default=31):
         help='k-mer size; default={d}'.format(d=default)
     )
 
+#https://stackoverflow.com/questions/55324449/how-to-specify-a-minimum-or-maximum-float-value-with-argparse#55410582
+def range_limited_float_type(arg):
+    """ Type function for argparse - a float within some predefined bounds """
+    min_val = 0
+    max_val = 1
+    try:
+        f = float(arg)
+    except ValueError:
+        raise argparse.ArgumentTypeError("Must be a floating point number")
+    if f < min_val or f > max_val:
+        raise argparse.ArgumentTypeError(f"Argument must be >{str(min_val)} and <{str(max_val)}")
+    return f
+
+
+def add_threshold_arg(parser, default=0.1):
```

Comment:
please make this `add_tax_threshold_arg` or something more specific to this PR, thanks!